ai_genomics/pipeline/validation/abstracts.py

```diff
@@ -0,0 +1,111 @@
+"""ai_genomics/pipeline/validation/abstracts.py
+Generates csv files of abstracts with some overlapping rows for validation.
+"""
+from ai_genomics import PROJECT_DIR, get_yaml_config
+from ai_genomics.utils.reading import make_path_if_not_exist
+from ai_genomics.getters.patents import get_ai_genomics_patents
+from ai_genomics.getters.openalex import (
+    _get_openalex_ai_genomics_abstracts,
+    _get_openalex_ai_genomics_works,
+)
+from ai_genomics.pipeline.validation.utils import (
+    generate_overlapping_samples,
+)
+import pandas as pd
+
+
+if __name__ == "__main__":
+
+    config = get_yaml_config(PROJECT_DIR / "ai_genomics/config/validation.yml")
+
+    out_dir = PROJECT_DIR / "inputs/validation"
+    make_path_if_not_exist(out_dir)
+
+    patent_drop_cols = [
```

Comment:
Yeah that's right. There was a rationale for this in an earlier version because I needed to retain certain columns for a later operation, but it is redundant now!